src/black_primer/primer.json

```diff
@@ -110,6 +117,13 @@
       "long_checkout": false,
       "py_versions": ["all"]
     },
+    "typeshed": {
```

Comment:
Nice! Didn't know type shed had python we would format!

Reply:
We can format stubs too! That makes typeshed a good way to test our stub formatting.